bin/identity-tp

```diff
@@ -30,7 +30,7 @@ sys.path.insert(0, os.path.join(
     os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
     'families', 'identity'))
 
-from sawtooth_identity.processor.main import main
+from sawtooth_identity_py.processor.main import main
```

Comment:
We should avoid naming packages based on the implementation language.  This should remain as `sawtooth_identity`, but probably needs to be organized at a higher level. 

Likewise, we should talk about whether or not we want to keep the python implementation at all.